view/stache/doc/can-import.md

```diff
@@ -48,6 +48,43 @@ Example:
 {{/eq}}
 ```
 
+## static & dynamic import
+### static
+These are imports that are direct dependencies of a template.
+
+Example:
+
+```
+<can-import from="mymodule"/>
+```
+which is equivalent to a ES6 import like:
+```
+import from "mymodule";
+```
+
+### dynamic
+These are conditional imports, things you only want to import in certain situations. Like described before in section "Progressive Loading".
+```
+<can-import from="components/foobar">
+  {{#if isResolved}}
+  <foobar/>
+  {{/if}}
+</can-import>
+```
+which is equivalent to a stealJS import like:
+```
+import loader from "@loader";
```

Comment:
ok.. i rewrite to `System.import`. i can leave also a note that `@loader` would be recommended way ;)